nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-client-service/src/main/java/org/apache/nifi/distributed/cache/client/CacheClientRequestHandler.java

```diff
@@ -86,5 +98,8 @@ public void invoke(final Channel channel, final OutboundAdapter outboundAdapter,
         channel.writeAndFlush(Unpooled.wrappedBuffer(outboundAdapter.toBytes()));
         channelPromise.awaitUninterruptibly();
         this.inboundAdapter = new NullInboundAdapter();
+        if (channelPromise.cause() != null) {
+            throw new IOException(channelPromise.cause());
```

Comment:
Recommend include a message with the exception:
```suggestion
            throw new IOException("Request invocation failed", channelPromise.cause());
```